packages/module/src/InvalidObject/InvalidObject.tsx

```diff
@@ -0,0 +1,47 @@
+import { Button, Title } from '@patternfly/react-core';
+
+import Icon404 from './icon-404';
+import React from 'react';
+import classNames from 'classnames';
+import { createUseStyles } from 'react-jss'
+
+const useStyles = createUseStyles({
+  "ins-c-component_invalid-component": {
+    display: "flex",
+    justifyContent: "center",
+    flexDirection: "column",
+    alignItems: "center",
+    textAlign: "center",
+    h1: {marginBottom: "30px"},
```

Comment:
We should be using the PF token values instead of constants where possible. Alternatively, we can use the PF classes like `pf-v5-u-mb-lg` etc.

Reply:
We should stick to tokens as it will be needed when theming is released.